File: Fortran/0140/0140_0000.f90
module mod
integer:: var
integer,parameter::var1=1
contains
subroutine sub
do i=1,var
end do
end subroutine
end module

use mod
integer:: var2
integer,parameter::var3=2
do i=1,10
end do
do i=1,10
end do
do i=1,10
end do
print *,'pass'
end

File: C/0140/0140_0000.c
#include <stdio.h>


int f;
int a[100];
int sub(b)
int b[100];
{
  int i;
  for (i=0; i<100; i++) {
    a[i]=b[i]*f;
  }
}
int main()
{
  int b[100];
  int sum;
  int i;

  for (i=0; i<100; i++) {
    b[i]=i;
  }
  f=b[1];
  sub(b);
  sum = 0;
  for (i=0; i<100; i++) {
    sum+=a[i];
  }
  printf("%d\n",sum);
}

